config/schemas/config.yaml

```diff
@@ -6634,6 +6651,17 @@ properties:
       extraIssuers:
         title: Extra Issuers
         type: array
+        items:
+          $ref: "https://github.com/datreeio/CRDs-catalog/raw/refs/heads/main/cert-manager.io/clusterissuer_v1.json"
```

Comment:
I am a little bit worried about the security and performance of fetching these over the Internet, which is why we haven't done it like this before. Not sure how to proceed with that.

Reply:
One way I looked at before was to pass referenced schemas via `yajsv -r path/to/schemas`, but at the time decided against because it made local references exceedingly verbose as they needed to be absolute URLs (matching the `$id` of the referenced schemas)